src/main/java/com/google/flourbot/entity/Macro.java

```diff
@@ -0,0 +1,33 @@
+package com.google.flourbot.entity;
+
+import java.util.Map;
+import java.lang.Object;
+
+import com.google.flourbot.entity.action.Action;
+import com.google.flourbot.entity.trigger.Trigger;
+
+public final class Macro {
```

Comment:
No major change except naming and changing variable to be private and immutable.